### PR TITLE
cremaker def files added

### PR DESCRIPTION
**Pull request by hyu7000**
cremaker M1, M2, S1, extruder, 200 bed, 220 bed obj file added

### CODE REVIEW

**Comment by hyu7000** [comment]
hello. 

some value is changed. could you check it out?

**Comment by nallath** [comment]
Sorry, CI is still failing.

**Comment by hyu7000** [comment]
some "default_value" was changed to "value".

If this keep failing, Could you tell me if there's anything i'm missing?

**Comment by konskarm** [comment]
Hi @hyu7000. Thank you for your contribution. Could you please provide links to the product pages of these printers? I cannot find their specifications anywhere.

**Comment by hyu7000** [comment]
https://www.facebook.com/cremaker/
cremaker's facebook is here. 

https://cafe.naver.com/cremaker
this link is korean user web comunity of cremaker

it's korean's company.

do you need any other imformation?

**Comment by hyu7000** [comment]
https://smartstore.naver.com/cremaker/products/5311082072

Additionally, this is shop link

File: resources/definitions/cremaker_common.def.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "acceleration_layer_0": { "value": 1000 },
+    "acceleration_print_layer_0": { "value": 1000 },
+    "acceleration_travel_layer_0": { "value": 1000 },
+    "retraction_amount": { "value": 1.2 },
+    "retraction_speed": { "value": 40 },
+    "retraction_combing": { "value": "Within Infill" },
+    "retraction_hop_enabled": { "value": true },
+    "retraction_hop_only_when_collides": { "value": true },
+    "retraction_hop": { "value": 0.3 },
+    "adhesion_type": { "value": "skirt" },
```
Comment (konskarm) [inline resources/definitions/cremaker_common.def.json:47]:
```suggestion
    "adhesion_type": { "value": "'skirt'" },
```

File: resources/definitions/cremaker_common.def.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "acceleration_wall": { "value": 800 },
+    "acceleration_wall_0": { "value": 800 },
+    "acceleration_wall_x": { "value": 800 },
+    "acceleration_travel": { "value": 1250 },
+    "acceleration_layer_0": { "value": 1000 },
+    "acceleration_print_layer_0": { "value": 1000 },
+    "acceleration_travel_layer_0": { "value": 1000 },
+    "retraction_amount": { "value": 1.2 },
+    "retraction_speed": { "value": 40 },
+    "retraction_combing": { "value": "Within Infill" },
```
Comment (konskarm) [inline resources/definitions/cremaker_common.def.json:43]:
```suggestion
    "retraction_combing": { "value": "'infill'" },
```

File: resources/definitions/cremaker_common.def.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "icon": "icon_ultimaker2",
+    "has_materials": true,
+    "machine_extruder_trains": {
+      "0": "cremaker_extruder_0"
+    }
+  },
+
+  "overrides": {    
+    "machine_heated_bed": { "default_value": true },
+    "machine_center_is_zero": { "default_value": false },
```
Comment (konskarm) [inline resources/definitions/cremaker_common.def.json:19]: This is already inherited as `false` from the `fdmprinter.def.json` so no need to include it here.
```suggestion
```

**Review by konskarm — CHANGES_REQUESTED** [review]
Some changes are needed for this to work. Currently, Cura crashes when adding a Cremaker printer because of those issues.

**Comment by hyu7000** [comment]
thank you for letting me know code to change.
changed some code. check it please.

**Comment by konskarm** [comment]
Everything seems to be working without issues now so I am merging these. the Joyplace Cremaker printers should appear in Cura 4.12 onwards.

**Pull request merged**